src/neo_usboard_v2_node.cpp

```diff
@@ -43,7 +46,49 @@ class ROS_Node : public neo_usboard_v2::ROS_NodeBase {
 	void handle(std::shared_ptr<const pilot::usboard::USBoardData> value) override
 	{
 		// TODO: convert and publish on ROS
-	}
+		neo_msgs::USBoardV2 usBoard;
+		int i = 0;
+		
+		for(const auto& sensor : config->sensor_config)
+		{
+			std_msgs::Header USRangeHeader;
+			sensor_msgs::Range USRangeMsg;
+			if(sensor.active)
+			{
+				usBoard.active[i] = 1;
+				// Creating rostopics only if they are active 
+				topicPub_USRangeSensor[i] = nh.advertise<sensor_msgs::Range>("/USBoardV2/Sensor"+std::to_string(i),1);
+				USRangeHeader.seq = 1; 				//uint32
+				USRangeHeader.stamp = ros::Time::now(); 		//time
+				USRangeHeader.frame_id = "usrangesensor"+std::to_string(i);		//string
+				USRangeMsg.header = USRangeHeader;
+				USRangeMsg.radiation_type = 0; 			//uint8   => Enum ULTRASOUND=0; INFRARED=1
+				USRangeMsg.field_of_view = 2.3; 			//float32 [rad]
+				USRangeMsg.min_range = 0.2; 				//float32 [m]
+				USRangeMsg.max_range = 3.0; 				//float32 [m]
+				USRangeMsg.range = value->sensor[i]; 	//float32 [cm] => [m]
```

Comment:
Both values are in meters right?

Reply:
Yes they are in meters ! It is in the comments